### DISCUSSION TITLE
Pagination overflow

### DISCUSSION THREAD

**Discussion by hasimpk** [gusgard/react-native-swiper-flatlist#132] [Q&A]
how can I handle pagination dots overflow?

<img width="364" alt="image" src="https://user-images.githubusercontent.com/18107674/155709068-3252aedc-9739-44c6-a0b0-f0045f8429b4.png">

**Comment by gusgard** [comment]
You could write your own Pagination component and use the Animated API https://github.com/gusgard/react-native-swiper-flatlist/blob/master/src/components/Pagination/Pagination.tsx
